app.js

```diff
@@ -9,9 +9,5 @@ const {
 
 app.get('/api/topics', getTopics);
 
-app.use((err, req, res, next) => {
-  if (err.status) res.status(err.status).send({ msg: err.msg });
-  else next(err);
-});
```

Comment:
No need to worry about it this time - but this is a change to the last branch rather than this one